rust/src/daemon.rs

```diff
@@ -70,7 +70,13 @@ pub(crate) fn deployment_populate_variant(
 
     dict.insert("pinned", &deployment.is_pinned());
     let unlocked = deployment.get_unlocked();
-    dict.insert("unlocked", &unlocked.to_string());
+    // Unwrap safety: This always returns a value
```

Comment:
For my own reference, as I've already seen this in quite a few occasions, is this basically happening because of https://github.com/gtk-rs/gir/issues/655?

Reply:
Yep see https://github.com/coreos/rpm-ostree/pull/2659#discussion_r593426451 above